packages/react-component-library/src/components/TabSet/TabSet.stories.tsx

```diff
@@ -9,6 +9,12 @@ export default {
   title: 'Tab Set',
   parameters: {
     actions: { argTypesRegex: '^on.*' },
+    docs: {
+      description: {
+        component:
+          'Use this component to alternate visible areas within the confines of a page.',
```

Comment:
Does alternate sound like there are only two tabs? Could go for something like 'switch between'.